src/explorer/SiteTreeItem.ts

```diff
@@ -195,6 +197,27 @@ export abstract class SiteTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         }
     }
 
+    public showCreatedOutput(actionContext: IActionContext): void {
+        const resource: string = this.root.client.isSlot ? 'slot' : 'web app';
+        const createdNewAppMsg: string = `Created new ${resource} "${this.root.client.fullName}": https://${this.root.client.defaultHostName}`;
+        ext.outputChannel.appendLine(createdNewAppMsg);
+        ext.outputChannel.appendLine('');
+
+        const viewOutput: MessageItem = { title: 'View Output' };
+        const deployButton: MessageItem = {
+            title: 'Deploy'
+        };
+
+        // Note: intentionally not waiting for the result of this before returning
+        window.showInformationMessage(createdNewAppMsg, deployButton, viewOutput).then(async (result: MessageItem | undefined) => {
+            if (result === viewOutput) {
```

Comment:
Can you add add a telemetry event just for this (similar to [this](https://github.com/microsoft/vscode-azurefunctions/blob/master/src/commands/deploy/notifyDeployComplete.ts#L26))

As mentioned [here](https://github.com/microsoft/vscode-azurefunctions/pull/1247#pullrequestreview-234134238), I don't want to add a "deploy" button to functions - but I could be convinced if telemetry shows a high percentage of people using the button in app service